crates/ironrdp-connector/src/connection.rs

```diff
@@ -65,15 +38,7 @@ pub enum ClientConnectorState {
     EnhancedSecurityUpgrade {
         selected_protocol: nego::SecurityProtocol,
     },
-    CredsspInitial {
-        selected_protocol: nego::SecurityProtocol,
-    },
-    /// At this point, user code may use `sspi::credssp::TsRequest::read_length` to find the correct frame length.
-    CredsspReplyNeeded {
-        selected_protocol: nego::SecurityProtocol,
-        credssp_client: Box<credssp::CredSspClient>,
-    },
-    CredsspEarlyUserAuthResult {
+    CredSsp {
```

Comment:
Similarly to above, the convention would be:
```suggestion
    Credssp {
```